### PR TITLE
Activation, path handling, notebook demos

### PR DESCRIPTION
**Pull request by CBroz1**
See parallel PRs for [element-deeplabcut](https://github.com/CBroz1/element-deeplabcut/pull/7) and [element-calcium-imaging](https://github.com/CBroz1/element-calcium-imaging/pull/2)

- Revised `ingest_session_scan` to handle relative paths
- Minor pyrat ingestion updates
   - add assertion for both credentials needed
   - revise confirmation message
- Clean up dlc video ingestion mechanism from `workflow-deeplabcut`, removing unnecessary functions
- `paths.py` revise relative path handling 
- `pipeline.py` - better handling of activation to remove need for previous `adamacs.utility.activate_many`

Note: there will be upstream conflicts due to [this SFB commit](https://github.com/SFB1089/adamacs/commit/b5fcb18b28d707533425f2c1bd9dccac5b84d2e1), but I propose we allow the SFB team to address these, as all items align with previous activation mechanism

### CODE REVIEW

File: adamacs/ingest/session.py [hunk 1]
```diff
@@ -1,101 +1,126 @@
...
-    if verbose: print(f"Number of scans found: {n_scans}")
+    n_scans = len(match_paths)
+    if verbose:
+        print(f'Number of scans found: {n_scans}')
 
-    basenames = [x.name for x in dirs]
-    
+    scan_pattern = "scan.{8}"
+    basenames = [x.name for x in match_paths]
     scan_keys = [re.findall(scan_pattern, x) for x in basenames]
```
Comment (ttngu207) [inline adamacs/ingest/session.py:43]: `re.findall` returns a list, resulting `scan_keys` being a list of list, is this intended? perhaps using `re.search` is more suitable?

File: adamacs/schemas/equipment.py [hunk 2]
```diff
@@ -4,9 +4,9 @@
 """
 import datajoint as dj
 from .. import db_prefix
-# db_prefix + 'equipment'
-schema = dj.schema()
+schema = dj.schema(db_prefix + 'equipment')
 
+__all__ = ['db_prefix']
```
Comment (ttngu207) [inline adamacs/schemas/equipment.py]:
```suggestion
__all__ = ['db_prefix', 'Equipment']
```

File: adamacs/schemas/subject.py [hunk 3]
```diff
@@ -97,8 +97,9 @@ class SubjectGenotype(dj.Manual):
     -> Subject
     -> Mutation
     ---
-    genotype        : enum('wt/wt', 'wt/tg', 'tg/wt', 'tg/tg')
+    genotype        : enum('wt/wt', 'wt/tg', 'tg/wt', 'tg/tg', 'cre+', '+/-')
```
Comment (ttngu207) [inline adamacs/schemas/subject.py:100]: It's worth considering a separate lookup table for `Genotype`, unless the list of genotypes above is fixed and likely to stay unchanged. Once the table is created, modifying the `enum` to add more entry is not very straightforward.

**Review by ttngu207 — APPROVED** [review]
Overall looks great. I left only a few minor comments

File: adamacs/schemas/subject.py (hunk 3 above)
Comment (CBroz1) [reply adamacs/schemas/subject.py:100]: In this case, the two additional items are only to ensure the PyRAT draft works on the test server. Based on previous conversations with Daniel, the four items initially included are the full set of items required for their project, motivating the `enum` data type

File: adamacs/ingest/session.py (hunk 1 above)
Comment (CBroz1) [reply adamacs/ingest/session.py:43]: Daniel's draft here captured each data type as a list, checked that that list only contained one unique value and, if so, proceeded with data insertion. This is not how I would have designed it from the ground up, but I conformed to the format thinking that there might be more reasons to do so than I currently understand.

**Pull request merged**